packages/flutter/lib/src/material/dialog.dart

```diff
@@ -229,9 +254,12 @@ class AlertDialog extends StatelessWidget {
     this.backgroundColor,
     this.elevation,
     this.semanticLabel,
+    this.insetPadding = const EdgeInsets.symmetric(horizontal: 40.0, vertical: 24.0),
```

Comment:
Wondering if this `const EdgeInsets` should be factored into a constant since its used twice.

Reply:
Good catch. Just updated to include this.